pages/api/play-attempt/index.ts

```diff
@@ -21,6 +21,7 @@ export async function getLastLevelPlayed(user: User) {
       $match: {
         userId: new ObjectId(user._id),
         updateCount: { $gt: 0 },
+        attemptContext: AttemptContext.UNBEATEN,
```

Comment:
This change could still show levels that you have already completed, vs before it would only say to continue playing if your latest playattempt is unbeaten

Reply:
isn't the conditional that i removed equal to adding this check here?